MasherJames/Ride-My-Way
# #158306883 Create a home page

What does this PR do?
creates a home page

 Description of Task to be completed?
Build the landing page of the Ride-My-Way app

 How should this be manually tested?
Clone the repo 
check for  ft-home-158306883 branch
navigate ito the ui folder
open index.html in a browser
 
 What are the relevant pivotal tracker stories?
#158306883

 Screenshots (if appropriate)
 
![screenshot from 2018-06-15 13-22-25](https://user-images.githubusercontent.com/32618673/41463525-72d9b13a-709f-11e8-87f0-f095f28167f8.png)
![screenshot from 2018-06-15 13-12-21](https://user-images.githubusercontent.com/32618673/41463530-75977290-709f-11e8-853a-b2efa1237a8a.png)
